doc/source/ray-core/fault_tolerance/gcs.rst

```diff
@@ -0,0 +1,71 @@
+.. _fault-tolerance-gcs:
+
+GCS Fault Tolerance
+===================
+
+Global Control Service (GCS) is a server that manages cluster-level metadata.
+It also provides a handful of cluster-level operations including :ref:`actor <ray-remote-classes>`, :ref:`placement group <ray-placement-group-doc-ref>` and node management.
```

Comment:
Maybe also mention node management?

Reply:
I already mentioned node management. Is there anything else you want to add?